README.md

```diff
@@ -1,3 +1,65 @@
 # cdn-configuration
 
-:construction: under construction :construction:
+## Overview
+
+https://github.com/ampproject/cdn-configuration stores the AMP versions that are being served by AMP CDNs in `configs/versions.json`. It also hosts the deployment and patching workflows for the release on-duty engineer.
+
+These workflows replace the actions on the internal release dashboard. It does not, however, replace the error monitoring systems. Please check the internal doc for links to monitor errors and performance.
+
+This README contains instructions for the releases on-duty engineer. To learn more about AMP releases, please see release documation at https://github.com/ampproject/amphtml/tree/main/docs.
+
+## Deploying and promoting a release
+
+Nightly releases are cut, built and deployed automatically.
+
+Releases are promoted to channels by updating `configs/versions.json` in a PR. These PRs are created on a schedule and assiged to the on-duty engineer, to be approved and merged.
+
+Once merged, the CDNs will pick up the new versions. It should take between 30 to 90 minutes for promotions to take effect.
```

Comment:
Cloudflare CDN is immediate (~1 minute), Google CDN is about 15-30 minutes (configs are faster than files) for existing releases and 30-90 for new releases. Do with this information what you will :)